cypress/e2e/live-pricing.cy.ts

```diff
@@ -0,0 +1,72 @@
+describe('Live pricing spec', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:8000/')
```

Comment:
The URL should be coming from env/base url
```suggestion
    cy.visit('http://localhost:8000/')
```

Reply:
Addressed @habib-deriv help to review again thanks 🙏